multiples.py

```diff
@@ -15,6 +16,14 @@ def good(i):
     return by_3 or by_5
```

Comment:
I suggest you simplify this code as well. Challenge: make it all fit on one line